ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/alter_table.md

```diff
@@ -36,17 +38,55 @@ ALTER TABLE `series` ADD INDEX `title_index` GLOBAL ON (`title`);
 
 You can specify any index parameters from the [`CREATE TABLE`](../create_table#secondary_index) command.
 
-Deleting an index:
+You can also add a secondary index using the {{ ydb-short-name }} CLI [table index](https://ydb.tech/en/docs/reference/ydb-cli/commands/secondary_index#add) command.
```

Comment:
All internal links need to be:

1. relative
2. lead to a markdown file (end with `.md`)

Reply:
ok, I will change that. I haven't noticed that, because it is just a copy-paste of the existing sentence